.ci/no-exception-test.sh

```diff
@@ -53,6 +53,27 @@ guava-with-sun-checks)
   rm sun_checks.*
   ;;
 
+openjdk14-with-checks-nonjavadoc-error)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: $CS_POM_VERSION
+  mkdir -p .ci-temp/
+  cd .ci-temp/
+  git clone https://github.com/checkstyle/contribution
+  cd ..
+  sed -i.'' 's/value=\"error\"/value=\"ignore\"/' \
+        .ci-temp/contribution/checkstyle-tester/checks-nonjavadoc-error.xml
+  cd .ci-temp/contribution/checkstyle-tester
+  sed -i '/  <!-- Filters -->/r ../../../.ci/openjdk14-excluded.files' checks-nonjavadoc-error.xml
+  export MAVEN_OPTS="-Xmx2048m"
+  groovy ./launch.groovy --listOfProjects ../../../.ci/openjdk-projects-to-test-on.config \
```

Comment:
@romani Note this is using launch and not diff.
Not expecting a change here.

Reply:
@nmancus1 , can we use diff.grovy here ?

Reply:
I can add the same `extraMvnOptions` command line arg to to `diff.groovy`, but since `diff.groovy` uses `launch.groovy` anyway, why not keep it as is and not have another utility to deal with?  Just curious.

Reply:
This goes back to discussion in main issue, ending with https://github.com/checkstyle/checkstyle/issues/8901#issuecomment-717596429 .

If we are to remove JXR from tester's POM to avoid telling it not to run anyways, we need to remove uses of `launch` as I think we are still relying on it generating HTML files since diff-report-tool is what generates the final HTML when `diff` is launched.

Reply:
So,  I should submit a PR to contribution to add `extraMvnOptions` to diff.groovy, then use that in this PR, right?

Reply:
I recommend to merge now and improve to use diff.groovy in next iteration/PR.

Reply:
Then this is resolved for this PR, I will just leave this open for follow up.

Reply:
@nmancus1 , please create issue on this , one day we will address it.

Reply:
https://github.com/checkstyle/checkstyle/issues/8960
https://github.com/checkstyle/contribution/issues/523